app/services/omniauth_authorizer.rb

```diff
@@ -44,4 +46,8 @@ def update_auth
   def update_session
     session[:omniauthed] = true
   end
+
+  def user
```

Comment:
Why create a new method? Why not just use a local above?  Especiually since we have methods in this class that use `user` as a local, that can be confusing.